crates/router/src/lib.rs

```diff
@@ -86,6 +86,7 @@ pub mod headers {
     pub const X_APP_ID: &str = "x-app-id";
     pub const X_REDIRECT_URI: &str = "x-redirect-uri";
     pub const X_TENANT_ID: &str = "x-tenant-id";
+    pub const X_WP_API_VERSION: &str = "WP-Api-Version";
```

Comment:
Why does this not start with `x`?

Reply:
Usage of `x` is not enforced, I guess? WP's docs mentions to use it as it is - https://developer.worldpay.com/products/access/payments/openapi